hudi-common/src/main/java/org/apache/hudi/metrics/JmxMetricsReporter.java

```diff
@@ -72,9 +70,29 @@ public JmxMetricsReporter(HoodieMetricsConfig config, MetricRegistry registry) {
     }
   }
 
+  private boolean isServerCreated() {
+    return jmxReporterServer != null;
+  }
+
+  private void initializeJmxReporterServer(String host, int[] ports) {
+    for (int port : ports) {
+      try {
+        jmxReporterServer = createJmxReport(host, port);
+        LOG.info("Started JMX server on port " + port + ".");
+        break;
+      } catch (Exception e) {
+        if (e.getCause() instanceof ExportException) {
+          LOG.info("Skip for initializing jmx port " + port + " because of already in use");
+        } else {
+          LOG.info("Failed to initialize jmx port " + port + ". " + e.getMessage());
+        }
```

Comment:
`LOG.info("Failed to initialize jmx port " + port + ". "， e);` ?

Reply:
emit log if all unknown error occur